### PR TITLE
Move filter decorator into FilterHelper

### PR DESCRIPTION
**Pull request by S2606**
Move filters decorator code to FilterHelper.py

Closes https://github.com/coala/coala/issues/4494

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by S2606** [comment]
@shreyans800755 Please review the pull request

File: coalib/parsing/filters/LanguageFilter.py [hunk 1]
```diff
@@ -1,7 +1,3 @@
-from coalib.parsing.filters.Filter import filter
-
-
-@filter
 def language_filter(bear, args):
     """
     Filters the bears by ``LANGUAGES``.
```
Comment (Nosferatul) [inline coalib/parsing/filters/LanguageFilter.py]: Something wrong here. I think you passed variable `bear` a tuple, but it shouldn't be a tuple.
`AttributeError: 'tuple' object has no attribute 'LANGUAGES'`

Comment (S2606) [inline coalib/parsing/filters/LanguageFilter.py]: @Nosferatul I think it may have passed by test file

Comment (Nosferatul) [inline coalib/parsing/filters/LanguageFilter.py]: then, I think you should change it.

**Comment by Makman2** [comment]
See https://github.com/coala/coala/issues/4494#issuecomment-320337889

**Comment by shreyans800755** [comment]
@S2606 
Tests are failing. Make sure it is passing. And still some work needs to be done. Have a look at discussion on the issue page.

**Comment by nvzard** [comment]
@Makman2  I think you need to close this PR

**Comment by Makman2** [comment]
Closing due to reassignment.

**Pull request closed without merging**